tools/dev-server/launcher_template.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# --- begin runfiles.bash initialization v2 ---
+# Copy-pasted from the Bazel Bash runfiles library v2.
```

Comment:
Can you expand this comment to say why this is copied?

Reply:
Yep. done!